### ISSUE TITLE
loss = 73.68

### ISSUE THREAD

**Issue by birdwcp** [Longqi-S/Focal-Loss#1]
when i change softmaxwithloss to focal-loss, in the beginning of training, the loss becomes 73.68
Is it normal?

**Comment by Longqi-S** [comment]
@birdwcp what's your class number and basic learning rate? Usually the loss is small. I have tested focal-loss in cifar10 and cifar100, and it works normally.

**Comment by birdwcp** [comment]
@Longqi-S 
classes  num :  2 ,    base learning rate : 0.001
in the beginning, the loss is 73.68,  after iteration 10 , Loss drop below 4.0

**Comment by Longqi-S** [comment]
@birdwcp 
Hello, usually the focal loss is smaller than normal softmax cross entropy loss in my opinion. Because it multiply a scalar less than 1. Maybe your learning rate is too small. In my experiments(cifar10 and cifar100), the loss is very small.

**Comment by birdwcp** [comment]
@Longqi-S 
how about the param alpha  and gamma?

**Comment by Longqi-S** [comment]
@birdwcp alpha is 0.5 and gamma is 1.

**Comment by HolmesShuan** [comment]
@Longqi-S Could you please share the final performance on CIfar100 using Focal Loss? According to my knowledge, Focal Loss may outperform the original SoftmaxWithLoss on hard tasks.
